### PR TITLE
FT2-40: Added a new single text field to the above form that will accept email id.

### PR DESCRIPTION
**Pull request by swarnavd**
User will enter email id and on submit, check if correct email id syntax has been used.
Show a message on successful email syntax or show an error message on the wrong syntax.

### CODE REVIEW

File: result.php [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+$dotenv = Dotenv::createMutable(__DIR__);
+$dotenv->load();
+$apiKey = $_ENV["apiKey"];
+
+// Declaring array for storing error messages in array format.
+$errorArray = [];
+$fullName = "";
+$trimmed = "";
+// Storing regex for checking.
+$namePattern = '/^[a-zA-Z]+$/';
```
Comment (innoDebrup) [inline result.php:13]:
```suggestion
$namePattern = '/^[a-zA-Z]+$/';
```

Comment (innoDebrup) [inline result.php:14]: `$namePattern = '/^[a-zA-Z]+$/';`
Explain more about this regular expression. What is it checking?

File: result.php [hunk 2]
```diff
@@ -0,0 +1,184 @@
...
+
+if ($_SERVER['REQUEST_METHOD'] == 'POST' && isset($_POST['Submit'])) {
+  $fName = $_POST['fName'];
+  $lName = $_POST['lName'];
+  $fullName = $fName . " " . $lName;
+  $trimmed = preg_replace($namePattern, " ", $fullName);
```
Comment (innoDebrup) [inline result.php:22]: `$trimmed = preg_replace($namePattern, " ", $fullName);`
Explain what we are doing with this statement.

**Review by innoDebrup — CHANGES_REQUESTED** [review]
- Please use PHP Alternate Control Syntax wherever applicable instead of using multiple PHP blocks in a random manner in HTML parts. It makes debugging problematic.

**Comment by swarnavd** [comment]
> * Please use PHP Alternate Control Syntax wherever applicable instead of using multiple PHP blocks in a random manner in HTML parts. It makes debugging problematic.

**Pull request closed without merging**